plugins/react-native.js

```diff
@@ -214,9 +214,10 @@ reactNativePlugin._normalizeData = function (data, pathStripRe) {
         data.culprit = normalizeUrl(data.culprit, pathStripRe);
     }
 
-    if (data.exception) {
+    var stacktrace = data.stacktrace || data.exception && data.exception.values[0].stacktrace;
```

Comment:
Maybe move this comment about `data.exception` guaranteeing other keys to be before this var decl; in its current position it doesn't obviously apply to `data.exception.values[0]`, which made me wonder if that was a safe access until I looked at the original where it did obviously apply.